src/PrestaShopBundle/Resources/views/Docs/cqrs-commands-list.md.twig

```diff
@@ -0,0 +1,16 @@
+{#@todo: where to put thi file? what about docs in other langs?#}
```

Comment:
> where to put this file

Putting Twig templates in Resources/views aims to enable Symfony to find them using namespace.
Here we dont need to do that so maybe we could keep it next to the Command ? in `src/PrestaShopBundle/Command` and we just use real filepath to give it to Twig

> what about docs in other langs

Devdocs is English only so no need for other langs 👍